requirements_dev.txt

```diff
@@ -6,6 +6,7 @@
 pytest~=6.2
 pytest-cov
 pytest-ordering~=0.6
+pytest-cases~=3.6.8
```

Comment:
Introduced pytest-cases which is extensively used to better organize unit tests in this PR with regard to the multi-session/contrasts/ground truth unit tests

Reply:
Being split into clean up PR.